client/src/Components/pokeDex/FavoritesList.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import {
+   Button,
+   List,
+   Box,
+   OrderedList,
+   ListItem,
+   IconButton,
+   HStack,
+   Heading,
+   Center,
+   Grid,
+   GridItem,
+} from "@chakra-ui/react";
+
+import { CloseButton } from "@chakra-ui/react";
+
+const FavoritesList = (props) => {
+   const { addFavorite } = props;
+   const [favoritesList, updateFavoritesList] = useState([]);
+
+   const selectBtn = async (e) => {
+      let id = e.currentTarget.id;
+      try {
+         const request = await axios
+            .delete(`http://localhost:4000/pokeDexFavorites?id=${id}`)
+            .then((response) => {
+               updateFavoritesList(response.data.rows);
+            })
+            .catch((error) => {
+               console.log(error);
+            });
+      } catch (e) {
+         console.log(e);
+      }
+   };
```

Comment:
Seems like there's a warning for this chunk of code... 
1. What does the warning mean to you?
2. What are the implications of the warning? What might happen if this is left as it is (hint: the **_real_** issue may be elsewhere)

Reply:
The above case leads React to compare the current render vs. the previous which will always be different, hence the **infinite loop**. The real issue was the state inside the `useEffect()` dependency. I have moved the `useState()` to a higher level (_App.js_), replaced the `useEffect()` dependency inside _FavoritesList.js_ with `updatedFavoritesList`, and had the backend send out the complete list after a specified action is completed.

Reply:
I have just commited the changes. 

Commit: https://github.com/maherhalabi/pokedex-picker/commit/9d81a4abeb3d63532a2f4e6f7dacbafedddfa3e5

Reply:
Awesome work on that fix; looks great!